pkg/terraform/exec/plugins/vendor/github.com/terraform-providers/terraform-provider-aws/aws/data_source_aws_eks_cluster_auth.go

```diff
@@ -0,0 +1,48 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/kubernetes-sigs/aws-iam-authenticator/pkg/token"
```

Comment:
@abhinavdahiya, here's where Kubernetes is getting pulled in, so I think [this][1] is not actually a problem.

[1]: https://github.com/openshift/installer/pull/1442/#discussion_r267145595